tests/utils/test_stage.js

```diff
@@ -183,7 +183,7 @@ stageAxisC = Vec3.multiply(TILE_UNIT, Quat.getUp(stageOrientation));
 //      hasLocalLights  - default is off
 //      hasSkybox       - default is on
 //      hasHaze         - default is off
-setupStage = function (flags, lifetime) {
+setupStage = function (flags, lifetime, originFrame) {
```

Comment:
Here i would recommend to put everything in a single struct (flags was meant for that)
instead of adding more parameters in the signature and having to set undefined in all the places where you want to specify the originFrame without changing the default lifetime

SO just add the expected parameters in the documentation above and explain what is the default behavior.